module/snv_workflow.nf

```diff
@@ -86,57 +120,37 @@ workflow workflow_extract_snv_annotations {
 
     main:
 
-    // We want to do all of the annotating with the GRCh38 / hg38 reference. If
-    // the liftover is going from h38 to hg19, defer until after annotations
```

Comment:
Yep! I had to retrain the models with an expanded training cohort so I took the opportunity to standardize when the annotations are being added for the two conversion directions. (Also updated the training so that we can use score v1.20 again)